src/components/Property/Types/NumericProperty/NumericProperty.tsx

```diff
@@ -0,0 +1,88 @@
+import { Flex, Group, Paper, Text } from '@mantine/core';
+
+import { usePropListeningState } from '@/api/hooks';
+import { NumericInput } from '@/components/Input/NumericInput/NumericInput';
+
+import { ConcretePropertyBaseProps } from '../../types';
+
+import { NumericPropertySlider } from './Slider/NumericPropertySlider';
+
+export interface NumericPropertyProps extends ConcretePropertyBaseProps {
+  setPropertyValue: (newValue: number) => void;
+  value: number;
+  additionalData: {
+    Exponent: number;
+    MaximumValue: number;
+    MinimumValue: number;
+    SteppingValue: number;
+  };
+}
+
+interface Props extends NumericPropertyProps {
+  isInt?: boolean;
+}
+
+export function NumericProperty({
+  disabled,
+  setPropertyValue,
+  value,
+  additionalData,
+  isInt = false
+}: Props) {
+  const { value: currentValue, setValue: setCurrentValue } =
+    usePropListeningState<number>(value);
+
+  const min = additionalData.MinimumValue;
+  const max = additionalData.MaximumValue;
+  const step = additionalData.SteppingValue;
+  const exponent = additionalData.Exponent;
+
+  // When no min/max is set, the marks for the slider cannot be nicely computed
+  const extent = max - min;
+  // @TODO (202503-03, emmbr) This should be handled a better way...
+  const maxAllowedExtentForSlider = 10e12;
```

Comment:
Should it be 1e12 or 1e13 instead? 10e12 looks weird to me or am I forgetting something about scientific notations